cl/people_db/management/commands/cl_import_judges.py

```diff
@@ -88,7 +102,11 @@ def import_fjc_judges(self, infile=None):
         df['Professional Career'].replace(to_replace=r';\sno', value=r', no',
                                             inplace=True, regex=True)
         for i, row in df.iterrows():
```

Comment:
Don't you need to have an `enumerate` around this to establish the index variable?

Reply:
My understanding of pandas 'iterrows' is that it returns an iterator containing the index of each row and the data in a row as a Series.  I don't think it's necessary here but I can re-write if you want.

Reply:
Right you are. I missed that this wasn't part of the diff.